iroh/src/magicsock.rs

```diff
@@ -2330,6 +2339,7 @@ impl Actor {
                         }) {
                         warn!(?discovery_item.node_addr, "unable to add discovered node address to the node map: {e:?}");
                     }
+                    self.msock.discovery_subscribers.send(discovery_item);
```

Comment:
I'm assuming the "lagging" concept will prevent this from ever locking/hanging correct?